go/connector/run.go

```diff
@@ -118,25 +136,43 @@ func Run(
 		"--log-driver", "none",
 		// Network to which the container should attach.
 		"--network", networkName,
-		// The entrypoint into a connector is always `flow-connector-proxy`,
-		// which will delegate to the actual entrypoint of the connector.
-		"--entrypoint", "/flow-connector-proxy",
-		// Mount the connector-proxy binary, as well as the output of inspecting the docker image.
-		"--mount", fmt.Sprintf("type=bind,source=%s,target=/flow-connector-proxy", tmpProxy.Name()),
-		"--mount", fmt.Sprintf("type=bind,source=%s,target=/image-inspect.json", tmpInspect.Name()),
+	}
+
+	if port != "" {
+		imageArgs = append(imageArgs,
+			"--publish",
+			fmt.Sprintf("%s:%s", port, port),
```

Comment:
Are we deliberately exposing this port outside the local machine, or should this maybe be `"localhost:%s:%s"` / `"127.0.0.1:%s:%s"`?